code/game/machinery/computer/communications.dm

```diff
@@ -133,7 +139,7 @@ var/list/shuttle_log = list()
 			setMenuState(usr,COMM_SCREEN_SECLEVEL)
 
 		if("newalertlevel")
-			if(issilicon(usr))
+			if(issilicon(usr) && !is_malf_owner(usr))
```

Comment:
Undocumented change and the ability to unilaterally set alert to green as long as you control the console is questionable